<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maquette</class>
 <widget class="QDialog" name="maquette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>911</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Voiture</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>891</width>
       <height>561</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="gestion">
      <attribute name="title">
       <string>Gestion</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>831</width>
         <height>511</height>
        </rect>
       </property>
       <property name="title">
        <string>Gestion de voiture :</string>
       </property>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>20</y>
          <width>421</width>
          <height>371</height>
         </rect>
        </property>
        <property name="title">
         <string> Affichage de Données :</string>
        </property>
        <widget class="QTableView" name="tab_voiture">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>401</width>
           <height>331</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>361</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajout des voitures</string>
        </property>
        <widget class="QPushButton" name="PB_Ajout">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>90</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>237</width>
           <height>136</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Matricule :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="Matricule"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Marque :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="Marque"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Genre :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="Genre"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>heure d'entrer :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="heure_E"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>heure de sortie :</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="heure_S"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>381</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Suppression de données :</string>
        </property>
        <widget class="QPushButton" name="PB_supp">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>30</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Matricule :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="M_supp">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>400</y>
          <width>186</width>
          <height>28</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="btn_edit">
         <property name="text">
          <string>Modiffier</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_afficher">
         <property name="text">
          <string>Afficher</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>321</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>Calcule nombre de voiture :</string>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Nombre de voitures :</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_calculer">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>70</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Calculer</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="res_calc">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>32</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="btn_stat">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>400</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>statistique</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>metier 3</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>metier 4</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>metier 5</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
